src/DynamoCoreWpf/ViewModels/Core/AnnotationExtension.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Dynamo.Models;
+
+namespace Dynamo.Wpf.ViewModels.Core
+{
+    /// <summary>
+    /// Extension method to check if a model exists in a group
+    /// </summary>
+    internal static class AnnotationExtensions
+    {
+        public static bool CheckIfModelExistsInAGroup(this IEnumerable<AnnotationModel> groups, Guid nodeGuid)
+        {
```

Comment:
Minor refactoring suggestion, this would read much more clearly if it was `ContainsModel`.